packages/react/src/Datepicker/Calendar/Calendar.js

```diff
@@ -0,0 +1,71 @@
+import React, { forwardRef, useState } from 'react';
+import Flex from '../../Flex';
+import DecadeView from './DecadeView';
+import MonthView from './MonthView';
+import YearView from './YearView';
+import Navigation from './Navigation';
+import {
+  toDate,
+} from '../shared/utils';
+
+const Calendar = forwardRef((
+  {
+    value, // string
+    calendarStartDay = 0, // 0 = Sunday, 1 = Monday, ...
+    locale = 'en-CA',
+    view = 'month', // one of 'month', 'year', 'decade'
+    onChange: onClickDay,
+    ...rest
+  },
+  ref,
+) => {
+  const activeStartDate = value ? toDate(value) : new Date();
+  const [currentView, setView] = useState(view);
+  const [currentDate, setActiveStartDate] = useState(activeStartDate);
+
+  return (
+    <Flex
+      ref={ref}
+      display="inline-flex"
+      direction="column"
```

Comment:
Rename `direction` to `flexDirection` to avoid naming conflict once we added the `direction` style prop.

See https://developer.mozilla.org/zh-CN/docs/Web/CSS/direction